intezer_sdk/endpoint_analysis.py

```diff
@@ -1,23 +1,50 @@
+import glob
+import json
+import os
+import pathlib
+from concurrent.futures import ThreadPoolExecutor
 from typing import List
 
+from intezer_sdk import consts
+from intezer_sdk._endpoint_analysis_api import EndpointScanApi
+from intezer_sdk.analysis import logger
```

Comment:
Let's create a logger for this module so we will be able to configure the logging for this specific module